### PR TITLE
Faq clean up

### PR DESCRIPTION
**Pull request by shaunmulligan**
Added a few troubleshooting bit and a some FAQs

### CODE REVIEW

File: pages/troubleshooting/faq.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ title: FAQs
 ---
 # Frequently Asked Questions
 
+* [Can I use Multiple containers?](#can-i-use-multiple-containers-)
+* [What version of Docker runs on the Devices?](#what-version-of-docker-runs-on-the-devices-)
+* [Why does `/data` report weird usage?](#why-does-data-report-weird-usage-)
 * [What NTP servers do resin.io devices use?](/troubleshooting/faq#what-ntp-servers-do-resin-io-devices-use-)
```
Comment (emirotin) [inline pages/troubleshooting/faq.md]: good time to change the rest of the links to be anchor-only

File: pages/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -5,33 +5,21 @@ title: Troubleshooting
...
-  * [The 'Start the Terminal Session' Button Doesn't Appear](/troubleshooting/troubleshooting#the-start-the-terminal-session-button-doesn-t-appear)
-  * [Clicking 'Open Terminal' in the Session Does Nothing](/troubleshooting/troubleshooting#clicking-open-terminal-in-the-session-does-nothing)
   * [Terminal Closes On Update](/troubleshooting/troubleshooting#terminal-closes-on-update)
   * [Can't Login to dashboard.resin.io](/troubleshooting/troubleshooting#can-t-login-to-dashboard-resin-io)
   * [I get `$'\r': command not found` when my device tries to run scripts](/troubleshooting/troubleshooting#i-get-r-command-not-found-when-my-device-tries-to-run-scripts)
+  * [Device keeps dropping off wifi](#device-keeps-dropping-off-wifi)
 * [Raspberry Pi](/troubleshooting/troubleshooting#raspberry-pi)
   * [My Device Doesn't Boot](/troubleshooting/troubleshooting#my-device-doesn-t-boot)
   * [Connectivity](/troubleshooting/troubleshooting#connectivity)
   * [SD Card Corruption](/troubleshooting/troubleshooting#sd-card-corruption)
```
Comment (emirotin) [inline pages/troubleshooting/troubleshooting.md]: same about updating the links

Comment (shaunmulligan) [inline pages/troubleshooting/troubleshooting.md]: true, ill update them now

**Comment by shaunmulligan** [comment]
@emirotin updated all the links

**Pull request merged**